src/features/helios/components/exoticShop/component/ExoticSeeds.tsx

```diff
@@ -77,7 +79,38 @@ export const ExoticSeeds: React.FC<Props> = ({ onClose }) => {
     );
   };
 
+  const onCaptchaSolved = async (captcha: string | null) => {
+    await new Promise((res) => setTimeout(res, 1000));
+
+    gameService.send("SYNC", { captcha });
+
+    onClose();
+  };
+
+  if (showCaptcha) {
+    return (
+      <CloudFlareCaptcha
+        action="exotic-seeds-sync"
+        onDone={onCaptchaSolved}
+        onExpire={() => setShowCaptcha(false)}
+        onError={() => setShowCaptcha(false)}
+      />
+    );
+  }
+
+  const restock = () => {
+    // setShowCaptcha(true);
```

Comment:
**question:** Was this commented out intentionally? At the moment I don't think the captcha will show.

Reply:
I copied over the code from other modals with sync logic.  At the moment the captcha will show, but if we decide to enable it in the future we can do it easily by commenting out `setShowCaptcha(true)`.  In the future we might want to consolidate the code into one place for better code management.